ccc/oci.py

```diff
@@ -78,7 +79,10 @@ def base_api_lookup(image_reference):
             return base_url
         return oc.base_api_url(image_reference)
 
-    routes = oc.OciRoutes(base_api_lookup)
+    routes = oc.OciRoutes(
+        base_api_url_lookup=base_api_lookup,
+        sanitise_tags=sanitise_tags,
```

Comment:
hmmm... do you think it is a good match to store this as part of the routes? For manifest-route, it _is_ part of URL. otoh, for `tags` method, it is not. -> I think it would be cleaner to either make this a parameter of oci-client, or wrap it into a second "OciOptions" cfg object (although that might be a bit too heavyweight)

it would also be nice to document semantics of this flag (also) at params for oci.client's initialiser (__init__)